Pod/Support/iOS/RunTestsTargetWithSlimPort

```diff
@@ -2,90 +2,77 @@
 MYPATH=$(dirname $0)
 source $1
 SLIM_PORT=$2
-
 APPLICATION_PATH=${TARGET_BUILD_DIR}/${EXECUTABLE_FOLDER_PATH}
 DEPLOYMENT_TARGET_VALUE=$( echo "$DEPLOYMENT_TARGET_SUGGESTED_VALUES" | rev | cut -d " " -f1 | rev )
-DEVICE_TYPE=iPhone
-
 
 get_app_executable_name() {
-	echo $(/usr/libexec/PlistBuddy -c "Print :CFBundleExecutable" "$APPLICATION_PATH/Info.plist")
+	echo $(defaults read $APPLICATION_PATH/Info CFBundleExecutable)
 }
 
 get_app_devicetypeid() {
 
-	local BOOTED_DEVICETYPE_ID=
+	local DEVICETYPE_ID=
 	local BOOTED_DEVICE_NAME=$(xcrun simctl list devices | grep "Booted" | cut -d "(" -f1 | awk '{gsub(/^ +| +$/,"")} {print $0}')
 	
 	if [ ! -z "$BOOTED_DEVICE_NAME" ]; then
-		BOOTED_DEVICETYPE_ID=$(xcrun simctl list devicetypes | grep "^$BOOTED_DEVICE_NAME (com.apple" | cut -d "(" -f2 | cut -d ")" -f1)
+		DEVICETYPE_ID=$(xcrun simctl list devicetypes | grep "^$BOOTED_DEVICE_NAME (com.apple" | cut -d "(" -f2 | cut -d ")" -f1)
 	else
-		BOOTED_DEVICETYPE_ID=$(xcrun simctl list devicetypes | grep "$DEVICE_TYPE" | tail -n1 | cut -d "(" -f2 | cut -d ")" -f1)
+		DEVICETYPE_ID=$(xcrun simctl list devicetypes | grep "$DEVICE_TYPE" | tail -n1 | cut -d "(" -f2 | cut -d ")" -f1)
 	fi
 	
-	echo "$BOOTED_DEVICETYPE_ID, $DEPLOYMENT_TARGET_VALUE"
+	if [ -z "$DEVICETYPE_ID" ]; then
+		log "Device Type Identifier For $DEVICE_TYPE Missing"
+		exit 1
+	fi
+
+	echo "$DEVICETYPE_ID, $DEPLOYMENT_TARGET_VALUE"
 }
 
 slim_system_wait() {
 	
 	local EXECUTABLE_PID=
 	local EXECUTABLE_NAME=$(get_app_executable_name)
 	
-	echo "[OCSP_RUN] Waiting for Slim Test System To Launch"
+	log "Waiting for Slim Test System To Launch"
 	while [ -z "$EXECUTABLE_PID" ] 
 	do
 		EXECUTABLE_PID=$(pgrep "$EXECUTABLE_NAME")
 	done
 	
-	echo "[OCSP_RUN] Slim System Detected ($EXECUTABLE_PID)"
+	log "Slim System Detected ($EXECUTABLE_PID)"
 	while [[ "$EXECUTABLE_PID" ]]
 	do
 		EXECUTABLE_PID=$(pgrep "$EXECUTABLE_NAME")
 		sleep 0.5
 	done
 }
 
-ios_sim_launch() {
-	
-	echo "[OCSP_RUN] == Start Slim Test Sytem (ios-sim version $(ios-sim --version), Node Version: $(node -v)) =="
-
-	ios-sim launch "$1" --devicetypeid "$2" --args "$3" --verbose &
-	
-	slim_system_wait
-}
-
 run_test_system() {
-
-	local DEVICE_TYPE_IDENTIFIER=$(get_app_devicetypeid)
-
-	echo "[OCSP_RUN] Info: Test System SDK Deployment Target: $DEPLOYMENT_TARGET_VALUE"
-	echo "[OCSP_RUN] Info: Test System Executable Folder Path: $APPLICATION_PATH"
-	echo "[OCSP_RUN] Info: Simulator Device Identifier: $DEVICE_TYPE_IDENTIFIER"
-
-	ios_sim_launch "$APPLICATION_PATH" "$DEVICE_TYPE_IDENTIFIER" "$SLIM_PORT"
-	return $?
+	log "Start Slim Test Sytem"
+	$MYPATH/simctl $APPLICATION_PATH $SLIM_PORT
+ 	if [ $? == 0 ]; then 
+	 	slim_system_wait; 
+	else 
+		log "**ABORT** (See Error Log)"
```

Comment:
This message could really help, if it could specify the location of the error log.